.github/workflows/releaseTrigger.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Release
+
+on:
+  create:
+    tags:
+      - 'releaseTrigger'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: releasing
+    steps:
+      - name: Wait for build to succeed
+        uses: jitterbit/await-check-suites@v1
+        with:
+          appSlugFilter: 'github-actions'
+          timeoutSeconds: '300'
+      - uses: actions/checkout@v2
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Perform Release
+        uses: eskatos/gradle-command-action@v1
+        env:
+          TRAVIS_BRANCH: master
+          TRAVIS_COMMIT_MESSAGE: <no commit message, not needed, we do not care about them, if we trigger or not>
+          TRAVIS_PULL_REQUEST: false
```

Comment:
The `TRAVIS_` is necessary because of ShipKit, correct? If so, can you add a comment here that clarifies that?

Reply:
done